controllers/renovate_util.go

```diff
@@ -320,15 +463,19 @@ func generateRenovateConfigForNudge(target updateTarget, buildResult *BuildResul
 
 	packageRules = append(packageRules, DisableAllPackageRules)
 	packageRules = append(packageRules, PackageRule{
-		MatchPackageNames:  matchPackageNames,
-		GroupName:          fmt.Sprintf("Component Update %s", buildResult.Component.Name),
-		BranchName:         fmt.Sprintf("konflux/component-updates/%s", buildResult.Component.Name),
-		CommitMessageTopic: buildResult.Component.Name,
-		PRFooter:           "To execute skipped test pipelines write comment `/ok-to-test`",
-		RecreateWhen:       "always",
-		RebaseWhen:         "behind-base-branch",
-		Enabled:            true,
-		FollowTag:          buildResult.BuiltImageTag,
+		MatchPackageNames:   matchPackageNames,
+		GroupName:           fmt.Sprintf("Component Update %s", buildResult.Component.Name),
+		BranchPrefix:        BranchPrefix,
+		BranchTopic:         buildResult.Component.Name,
+		CommitMessageTopic:  buildResult.Component.Name,
+		PRFooter:            "To execute skipped test pipelines write comment `/ok-to-test`",
+		PRHeader:            fmt.Sprintf("image created from '%s'", gitRepoAtShaAnnotation),
+		RecreateWhen:        "always",
+		RebaseWhen:          "behind-base-branch",
+		Enabled:             true,
+		FollowTag:           buildResult.BuiltImageTag,
+		CommitMessagePrefix: target.ComponentCustomRenovateOptions.CommitMessagePrefix,
+		CommitMessageSuffix: fmt.Sprintf(", image created from '%s' \n %s", gitRepoAtShaAnnotation, target.ComponentCustomRenovateOptions.CommitMessageSuffix),
```

Comment:
Cross-posting from Slack - can we break the commit message to multiple lines to avoid breaking gitlint users? Something like this?

```
<Normal commit message>

Image created from:
<git repo @ commit sha>
```

Reply:
well renovate is ignoring even new line which I have there, I think it basically works just with single line commit

Reply:
Maybe we want `CommitBody` rather than `CommitMessageSuffix`? https://docs.renovatebot.com/configuration-templates/#commit-message

Reply:
using commit Body now